src/NewsService.Data.Provider.MsSql.Ef/Migrations/20210812164100_AddServicePropertiesToDb.cs

```diff
@@ -0,0 +1,61 @@
+﻿using LT.DigitalOffice.NewsService.Models.Db;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+
+namespace LT.DigitalOffice.NewsService.Data.Provider.MsSql.Ef.Migrations
+{
+    [DbContext(typeof(NewsServiceDbContext))]
+    [Migration("20210812164100_AddServicePropertiesToDb")]
```

Comment:
No need to write "toDb", migration works with db anyway.

Reply:
done